Content.Server/Whitelist/WhitelistCommands.cs

```diff
@@ -31,19 +33,19 @@ public override async void Execute(IConsoleShell shell, string argStr, string[]
         var playtime = IoCManager.Resolve<PlayTimeTrackingManager>();
```

Comment:
```
var db = IoCManager.Resolve<IServerDbManager>();
var loc = IoCManager.Resolve<IPlayerLocator>();
```

I think we can remove these now (a couple lines up) - they aren't being used anymore.

These were removed in the upstream commit, but not in our cherry pick
- upstream commit: `69b3e355e43f235e088c504952addef6b5598def`
- cherry pick commit from this PR: `e5cef17c82c42f4ac2deb6f529cc1f5391e902e3`